nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -24,22 +25,32 @@ using std::placeholders::_1;
 namespace opennav_docking
 {
 
-DockingServer::DockingServer(const rclcpp::NodeOptions & options)
-: nav2_util::LifecycleNode("docking_server", "", options)
+DockingServer::DockingServer(rclcpp::NodeOptions options)
+: nav2_util::LifecycleNode("docking_server", "",
+    options.automatically_declare_parameters_from_overrides(true))
```

Comment:
Why the overrides? That's a big change - I think that should be reverted

Reply:
With the override, you are allowed to read the ````dock_backwards```` parameter from the yaml file without having previously declared it. This way, if the node has the parameter, it is because it came from an old yaml configuration.